src/main/pages/workspace-admin/recipes.adoc

```diff
@@ -11,42 +11,56 @@ folder: workspace-admin
 [id="supported-recipe-formats"]
 == Supported Recipe Formats
 
-Depending on the infrastructure, Che supports the following recipes out of the box (default Che build):
+Depending on the infrastructure, Che supports the following default recipe formats:
 
 [cols=",,,,",options="header",]
 |===
-|*Infrastructure* |*Docker Image* |*Dockerfile* |*Composefile* |*Kubernetes yaml*
+|*Infrastructure* |*Docker-formatted container image* |*Dockerfile* |*Composefile* |*Kubernetes YAML*
 |*Docker* |Supported |Supported |Supported |Not supported
 |*OpenShift* |Supported |Not supported |Not supported |Supported
 |===
 
-[id="docker-image"]
-== Docker Image
+[id="docker-formatted-container-image-requirements-and-limitations"]
+== Docker-formatted container image requirements and limitations
 
-DockerImage recipe type implies that Che will pull an image from a Docker registry or use the one that is local and run it (in case of Docker) or create a pod that references this image in container spec. There are certain requirements and limitations for a Docker image to be a workspace machine:
+The Docker-formatted container image recipe pulls an image from a Docker registry or uses the local image.  The recipe then runs the image and creates a pod that references this image in the container specification. The following are Docker-formatted container image requirements and limitations for a workspace machine:
 
-1.  Non-terminating `CMD` or `ENTRYPOINT`. If you use your custom image, make sure it has something like `tail -f /dev/null` as one of its main processes.
-2.  No processes and operations with `sudo` in `CMD` (OpenShift only, see link:openshift-config.html#enable-ssh-and-sudo[Enable sudo])
-3.  OpenShift only - it is recommended to use https://github.com/eclipse/che-dockerfiles/tree/master/recipes/stack-base[Che base stacks] that are certified to run OK on OpenShift. You can also build your own image, but inherit from one of base stacks.
+1.  Use a non-terminating `CMD` or `ENTRYPOINT`. If you use a custom image, ensure that `tail -f /dev/null` is one of the main processes.
```

Comment:
`tail -f /dev/null` does not _have_ to be a main process -- it's an example of a non-terminating command which will run indefinitely. There are multiple commands that could be used.